media/js/firefox/new/experiment-win10-taskbar-funnelcake.js

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function(Mozilla) {
+    'use strict';
+
+    var ua = navigator.userAgent;
+    var isLikeFirefox = /Iceweasel|IceCat|SeaMonkey|Camino|like\ Firefox/i.test(ua);
+    var isFirefox = /\s(Firefox|FxiOS)/.test(ua) && !isLikeFirefox;
+    var isMobile = /^(android|ios|fxos)$/.test(window.site.platform);
```

Comment:
I'm not sure we need this check (and below), as we're already checking that `window.site.platform` is exactly `windows` - which infers *not* mobile...right?

Reply:
good catch, will remove